### ISSUE TITLE
Doesn't work for links to elements

### ISSUE THREAD

**Issue by AntonC9018** [jghauser/follow-md-links.nvim#21]
`link_to_file.md#Text` doesn't work, while `link_to_file.md` works file

**Comment by jghauser** [comment]
Thanks for the issue report! This feature is indeed missing from the plugin. Just to give you a heads-up, I'm not sure when I'll have the time to get to this. I would be happy to accept PRs though! :smiley:

**Comment by AntonC9018** [comment]
Yeah, I think the plugin idea in general doesn't work for me. I want the same binding to figure out what type of link any link is, and open it up either in the editor or on the web. I also want it to work in any context, rather than just in markdown. So you won't get a PR from me :)
